### ISSUE TITLE
[debian] please move videobridge.rc to /etc

### ISSUE THREAD

**Issue by DanScharon** [jitsi/jitsi-videobridge#218]
At the moment videobridge.rc resides within /usr/share/jitsi-videobridge/lib which not only violates FHS but it also means that any changes made to it are overwritten with every package upgrade.
videobridge.rc should therefore be moved to /etc/jitsi/videobridge and loaded by jvb.sh accordingly (or even better: its contents merged with /etc/jitsi/videobridge/config).

Side question: why is the default value of VIDEOBRIDGE_MAX_MEMORY 3072m? Would you expect some side effects with low values like 512m and do you recommend a certain minimum of the max value?

**Comment by damencho** [comment]
Yeah I think this is correct. Maybe we just need to keep compatibility with old one.

**Comment by sanvila** [comment]
> or even better: its contents merged with /etc/jitsi/videobridge/config

Yes, please!

> Maybe we just need to keep compatibility with old one.

Those who install the Debian packages and upgrade them will have already lost their changes several times (and presumably, already found other ways to pass extra options).

So, I believe that completely dropping `videobridge.rc` at this point should be safe.

Ideally, I'd like to see the setting `-Xmx3072` to be explicit in `/etc/jitsi/videobridge/config`. That way everybody would know easily how to change the memory setting without adding extra documentation for that.

**Comment by sanvila** [comment]
This is just a hint of what I would do and it's completely untested. This is public domain so please feel free to take this and put your name in the Author if required, as I don't plan to sign contributor paper for now.

https://github.com/sanvila/jitsi-videobridge/commit/84b6774be6c7259e2f379c10f1db88558f53abb6

Additionaly, postinst should add JVB_MEMORY to the config file on upgrades if it's not present, but I believe somebody else will figure out how to do that.
